indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/ec2/EC2AutoScaler.java

```diff
@@ -70,6 +71,15 @@ public EC2AutoScaler(
     this.envConfig = envConfig;
     this.amazonEC2Client = amazonEC2Client;
     this.config = config;
+
+    final String regionName = envConfig.getRegion();
```

Comment:
per docs in AmazonEC2:


```<b>This method is not threadsafe. A region should be configured when the
     * client is created and before any service requests are made. Changing it
     * afterwards creates inevitable race conditions for any service requests in
     * transit or retrying.</b>```


Can this be moved to `io.druid.guice.AWSModule#getEc2Client`?